.gitignore

```diff
@@ -14,3 +14,6 @@ npm-debug.log*
 
 .node-version
 .vscode
+
+# For webstorm IDE
+./idea/*
```

Comment:
Please use your own [global excludes](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c) instead of adding them to the project

Reply:
Done. This felt kinda crummy, thanks for the pointer.